### DISCUSSION TITLE
Big difference of error between train and test.

### DISCUSSION THREAD

**Discussion by B-C-WANG** [deepmodeling/deepmd-kit#6] [Q&A]
After I have trained 100000 steps using a smooth model, the l2 test error is about 1.58e-03, that's wonderful, but when I run dp_test, the error of energy is about 0.4, then I use the code from your "test.py" to predict the energy of train set "set.001", the l2 error is 0.153, here is a fig of real energy to predicted energy:
![qq 20180724153506](https://user-images.githubusercontent.com/25495799/43125137-b97b2364-8f5b-11e8-93a5-b10910e507e5.png)
I don't know why the error of train are so different from test, can you figure it out how to reduce the test error?

**Comment by amcadmus** [comment]
Could you please check the standard deviation of forces in the data? The error 0.15 eV/A looks not small. How many atoms are in your system? The energy error is normalized by the number of atoms in the system, therefore, it looks much smaller than the tested energy error that is not normalized. We would make the test more clear.
